doc/en/usage.rst

```diff
@@ -174,7 +174,11 @@ This will add an extra property ``example_key="1"`` to the generated
 
 .. code-block:: xml
 
-    <testcase classname="test_function" example_key="1" file="test_function.py" line="0" name="test_function" time="0.0009">
+    <testcase classname="test_function" file="test_function.py" line="0" name="test_function" time="0.0009">
+      <properties>
```

Comment:
I have read [1] carefully, and it seems like 'properties' element is allowed only in 'testsuite' ... that means that my patch actually doesn't align with junit results definition :disappointed: .
I was convinced that it does, because we use plugin on Jenkins which process data from properties inside of testcase element. Adding such ability to pytest-junitxml plugin was main reason behind this patch.

[1] http://help.catchsoftware.com/display/ET/JUnit+Format